model.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+import pandas as pd
+import numpy as np
+
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.ensemble import GradientBoostingClassifier
+from xgboost import XGBClassifier
+
+
+# TODO: implement model to try:
+#          try: SVM, Gradient Boosted Trees, XGBoost
+
+class Model(BaseEstimator, ClassifierMixin):

```

Comment:
Why implementing a model abstract class, if you use sklearn models where all this methods are implemented?
If you'll want to implement a model that sklearn don't have... You could implement it with this functions...